src/main/secure-local-store.ts

```
@@ -0,0 +1,49 @@
+      // Decrypt the data
+      const decrypted = safeStorage.decryptString(encryptedData);
+
+      // Parse the decrypted JSON string
+      const data = JSON.parse(decrypted);
+      return data[key];
+    } catch {
+      return undefined;
+    }
+  }
+
+  set(key: string, value: any): void {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)